File: app/src/main/java/com/example/android/newsapp/refactor/repository/remote/api/NewsService.java
package com.example.android.newsapp.refactor.repository.remote.api;

import com.example.android.newsapp.refactor.model.News;

import retrofit2.Call;
import retrofit2.http.GET;

public interface NewsService {

    //FIXME implement the search
    @GET("/search?maxResults=10&order-by=newest&api-key=test&q=")
    Call<News> getNews();
}

File: app/src/main/kotlin/com/example/android/newsapp/repository/api/NewsService.kt
package com.example.android.newsapp.repository.api

import com.example.android.newsapp.model.News
import retrofit2.Call
import retrofit2.http.GET

interface NewsService {

    @GET("/search?maxResults=10&order-by=newest&api-key=test&q=")
    fun getNews(): Call<News>
}
